src/Bicep.Decompiler/TemplateDecompiler.cs

```diff
@@ -19,45 +19,42 @@ namespace Bicep.Decompiler
 {
     public static class TemplateDecompiler
     {
-        private static Uri ChangeExtension(Uri prevUri, string newExtension)
-        {
-            var uriString = prevUri.ToString();
-            var finalDot = uriString.LastIndexOf('.');
-            uriString = (finalDot >= 0 ? uriString.Substring(0, finalDot) : uriString) + $".{newExtension}";
-
-            return new Uri(uriString);
-        }
-
         public static (Uri entrypointUri, ImmutableDictionary<Uri, string> filesToSave) DecompileFileWithModules(IResourceTypeProvider resourceTypeProvider, IFileResolver fileResolver, Uri jsonUri)
         {
-            var decompileQueue = new Queue<Uri>();
+            const string bicepExtension = "bicep";
```

Comment:
This might also fit into the `PathHelper` class - I notice we've got a `TemplateOutputExtension` const in there as well.

Reply:
Done.